pydatastructs/miscellaneous_data_structures/tests/test_queue.py

```diff
@@ -1,9 +1,18 @@
-from pydatastructs.miscellaneous_data_structures import Queue
+from pydatastructs.miscellaneous_data_structures import (
+    Queue, ArrayQueue, LinkedListQueue)
 from pydatastructs.utils.raises_util import raises
 
 def test_Queue():
+    q1 = Queue(implementation='array', items=[0])
+    q1 = Queue()
+    q1 = Queue(implementation='linked_list')
```

Comment:
Remove this function.

Reply:
this function checks for implementations of the abstract datatype. If changes are made which are supposed to be common in implementations the other developer can just make changes here.

Reply:
Well this function has no `assert` so nothing is checked here.

Reply:
sure will take care of that.